otter/test/convergence/test_gathering.py

```diff
@@ -213,85 +221,87 @@ class GetScalingGroupServersTests(SynchronousTestCase):
     """
 
     def setUp(self):
-        """Save basic reused data."""
```

Comment:
Oh. I am sorry I removed it. I don't know why I did that. Thanks so much for catching this. I'll add the tests. I guess the function name change caused the confusion.